kubeadm-config.yaml

```diff
@@ -1,31 +1,47 @@
----
-apiVersion: kubeadm.k8s.io/v1beta3
+apiVersion: kubeadm.k8s.io/v1beta4
 kind: InitConfiguration
+localAPIEndpoint:
+  bindPort: ${U7S_PORT_KUBE_APISERVER}
 ---
-apiVersion: kubeadm.k8s.io/v1beta3
+apiVersion: kubeadm.k8s.io/v1beta4
 kind: ClusterConfiguration
-networking:
-  serviceSubnet: "10.96.0.0/16"
-  podSubnet: "10.244.0.0/16"
-controlPlaneEndpoint: "${U7S_NODE_NAME}:6443"
+controlPlaneEndpoint: "${U7S_NODE_NAME}:${U7S_PORT_KUBE_APISERVER}"
 apiServer:
-  extraArgs:
-    advertise-address: "${U7S_HOST_IP}"
-    cloud-provider: external
-    # Default: "Hostname,InternalDNS,InternalIP,ExternalDNS,ExternalIP"
```

Comment:
This comment seems lost

Reply:
Added back: https://github.com/rootless-containers/usernetes/pull/346/commits/1166cf2f8b034e3bff515a7a1b1abab1f3ac91c5